templates/etc/icinga2/features-available/api.conf.j2

```diff
@@ -7,8 +7,8 @@
  */
 
 object ApiListener "api" {
-  //accept_config = false
-  //accept_commands = false
+  accept_config = true
```

Comment:
Can you add a comment to both flags why the API needs to accept `config` and `commands`?

Reply:
to accept configs and command over the API. I don't think this needs more clarification.